packages/editor/src/browser/editor-widget.ts

```diff
@@ -28,6 +29,10 @@ export class EditorWidget extends BaseWidget implements SaveableSource {
         return this.editor.document;
     }
 
+    get targetUri(): URI | undefined {
+        return this.editor.targetUri;
```

Comment:
why not just use `this.editor.uri`?

Reply:
We cannot use just `this.editor.uri` here because we can have different editors here and each of them could have its own underlying model. We have to delegate this work to it.